mkosi/__init__.py

```diff
@@ -4522,6 +4581,16 @@ def create_parser() -> ArgumentParserMkosi:
         help="If specified, the container/VM is run with a temporary snapshot of the output image that is "
         "removed immediately when the container/VM terminates",
     )
+    group.add_argument(
+        "--ssh", action=BooleanAction, help="Setup SSH access from the host to the final image via `mkosi ssh`"
```

Comment:
I'm not entirely sure, but I stumble over setup as a verb
```suggestion
        "--ssh", action=BooleanAction, help="Set up SSH access from the host to the final image via `mkosi ssh`"
```